package/gluon-l3roamd/files/etc/modules.d/gluon-l3roamd

```diff
@@ -0,0 +1 @@
+cfg80211
```

Comment:
now I remember. I included this file to have l3roamd start up even in the x86 image where this module is not automatically loaded. Also l3roamd will not work without this module yet.
That means: For now I will move the file into the l3roamd-package. Once the PR is confirmed working and merged I will remove the modules-file.